README.md

```diff
@@ -2397,6 +2397,45 @@ Other Style Guides
       'Batman',
       'Superman',
     ];
+    
+    // bad
+    function createHero(
+      firstName,
+      lastName,
+      inventorOf
+    ) {
+      // does nothing
+    }
+    
+    // good
+    function createHero(
+      firstName,
+      lastName,
+      inventorOf,
+    ) {
+      // does nothing
+    }
+    
+    // good
+    function createHero(firstName,
+                        lastName,
```

Comment:
It's a bad example for our styleguide - that style is absolutely not permitted, even if `comma-dangle` works with it. Other linter rules should be forbidding it.